PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalVaultRequest.kt

```diff
@@ -33,8 +33,8 @@ class PayPalVaultRequest
 @JvmOverloads constructor(
     override val hasUserLocationConsent: Boolean,
     var shouldOfferCredit: Boolean = false,
-    var recurringBillingDetails: PayPalRecurringBillingDetails? = null,
-    var recurringBillingPlanType: PayPalRecurringBillingPlanType? = null,
+    override var recurringBillingDetails: PayPalRecurringBillingDetails? = null,
+    override var recurringBillingPlanType: PayPalRecurringBillingPlanType? = null,
```

Comment:
We will also need to add these to the `PayPalCheckoutRequest` to enable this for the required flow